docs/next/guides/columns/column-width.md

```diff
@@ -68,7 +68,7 @@ const hot2 = new Handsontable(container, {
 
 ## Setting the column width using a function
 
-The column width can be set using a function. In this example, the size of all columns is set using a function by taking a column `index` (1, 2 ...) and multiplying it by `40px` for each consecutive column. 
+The column width can be set using a function. In this example, the size of all columns is set using a function by taking a column `index` (1, 2 ...) and multiplying it by `40px` for each consecutive column.
```

Comment:
The column width can be set using a function.  > please remove.

Reply:
Done in 4750b948fce55293009c9adc0a2ee379ad7d69e9